go.mod

```diff
@@ -54,6 +53,8 @@ replace (
 	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
 	github.com/mattn/go-sqlite3 => github.com/mattn/go-sqlite3 v1.10.0
 	golang.org/x/text => golang.org/x/text v0.3.3 // Required to fix CVE-2020-14040
+	sigs.k8s.io/kubebuilder/v3 => github.com/camilamacedo86/kubebuilder/v3 v3.0.0-20210920222540-6766994bb683
```

Comment:
/hold

we need to merge https://github.com/kubernetes-sigs/kubebuilder/pull/2340
and then remove this line